pkg/api/componentreadiness/component_report.go

```diff
@@ -1102,7 +1099,7 @@ func (c *componentReportGenerator) matchBaseRegression(testID crtype.ReportTestI
 		baseRegression = regressionallowances.IntentionalRegressionFor(baseRelease, testID.ColumnIdentification, testID.TestID)
 
 		// This could go away if we remove the option for ignoring fallback
-		if baseRegression != nil && baseRegression.PreviousPassPercentage() > getTestStatusSuccessRate(baseStats) {
+		if baseRegression != nil && baseRegression.PreviousPassPercentage(c.FlakeAsFailure) > c.getTestStatusPassRate(baseStats) {
```

Comment:
Shouldn't we update places like these as well so that flakes can be counted as failures when c.FlakeAsFailure is enaled?

https://github.com/openshift/sippy/blob/b218d0b853064bf3c54625f482ad1940473d5b09/pkg/api/componentreadiness/component_report.go#L1466

https://github.com/openshift/sippy/blob/b218d0b853064bf3c54625f482ad1940473d5b09/pkg/api/componentreadiness/component_report.go#L1458

Reply:
Ahh, nevermind so you are separating out the success, failure and flake and then checking the flag in getPassRate